test/controllers/easymon/checks_controller_test.rb

```diff
@@ -23,7 +23,7 @@ class ChecksControllerTest < ActionController::TestCase
 
     test "index when a check fails" do
       Easymon::Repository.add("database", Easymon::ActiveRecordCheck.new(ActiveRecord::Base))
-      Easymon::Repository.add("redis", Easymon::RedisCheck.new(YAML.load_file(Rails.root.join("config/redis.yml"))[Rails.env].symbolize_keys))
+      Easymon::Repository.add("redis", Easymon::RedisCheck.new(YAML.load_file(Rails.root.join("config/redis.yml"), aliases: true)[Rails.env].symbolize_keys))
```

Comment:
Psych >= 4.0 aliases load to safe_load, which requires alias support to be manually enabled.